data/sql_updates/create_linkage_names_view.sql

```diff
@@ -1,42 +1,13 @@
-drop materialized view if exists ofec_name_linkage_mv_tmp;
-create materialized view ofec_name_linkage_mv_tmp as
+drop materialized view if exists ofec_cand_cmte_linkage_mv_tmp;
```

Comment:
If we want to rename the table, do we want to rename the file `create_cand_cmte_linkage_view.sql`?